packages/next/src/client/components/react-dev-overlay/internal/components/copy-button/index.tsx

```diff
@@ -0,0 +1,42 @@
+export function CopyButton({
+  label,
+  content,
+  ...props
+}: {
+  label?: string
+  content: string
+  [key: string]: any
+}) {
```

Comment:
If we're forwarding any props here, I think we should make sure they're valid for a `span`.

```suggestion
export function CopyButton({
  label,
  content,
  ...props
}: React.HTMLProps<HTMLSpanElement> & {
  label?: string
  content: string
}) {
```